4_params_munge/src/munge_secchi_model_data.R

```diff
@@ -1,19 +1,18 @@
-munge_daily_secchi <- function(in_ind, out_ind, dow_crosswalk_ind) {
+munge_daily_secchi <- function(out_ind, kw_files_zip_ind) {
 
-  dowfile <- sc_retrieve(dow_crosswalk_ind, remake_file = '1_crosswalk_fetch.yml')
-  dow2nhd <- get(load(dowfile))
-  dow2nhd$DOW <- as.character(dow2nhd$dowlknum)
-  dow2nhd <- distinct(dow2nhd)
+  cwd <- getwd()
+  on.exit(setwd(cwd))
```

Comment:
It doesn't look like you ever change the current working directory. Are these two steps necessary?